site/docs/foundations/typography.mdx

```diff
@@ -5,101 +5,97 @@ sidebar:
 
 <Image
-  src="/img/foundations/typography-weight.png"
+  src="/img/foundations/typography-fontweight.png"
+  srcDark="/img/foundations/typography-fontweight-dark.png"
   alt="Example of small, default and strong font weights"
 />
 
-## Type hierarchy
+## Hierarchy
 
-Type hierarchy arranges text elements by varying font sizes, weights, and styles. This guides the reader's attention and conveys content structure.
+Salt offers a series of text styles designed to create a visual hierarchy through typographic design. Each text style is dependent on the applied theme. By effectively utilizing Salt's styles, you can guide the reader's focus and clearly convey the content's structure.
 
-| Text style | Purpose                                                                                                                     |
-| ---------- | --------------------------------------------------------------------------------------------------------------------------- |
-| Display    | Display styles are primarily for representing key metrics and mastheads                                                     |
-| Headings   | Headings separate sections of content into varying levels                                                                   |
-| Body       | Body is the default style for most components and text blocks                                                               |
-| Label      | Labels display a small amount of text that helps people understand the current context of a component                       |
-| Help       | Help text provides additional information or instructions, such as accepted data formats for input or file types for upload |
+### Font size
 
-<Image
-  src="/img/foundations/typography-headers.png"
-  alt="Example of Open Sans and Amplitude font"
-/>
+Font size is determined by the selected text style and the [density](https://www.saltdesignsystem.com/salt/foundations/density). Density influences how much content fits on a screen by adjusting the size and spacing of components. As density changes, font size adjusts accordingly to maintain proportionate and harmonious design elements.
+
+### Line height
+
+Line height controls the space between baselines in a block of text and is proportional to the text size. It's set to 1.3 times the font size across all styles and is influenced by the selected density.
 
 <Image
-  src="/img/foundations/typography-style-body-label.png"
-  alt="Graphical representation of Body and Label styles"
+  src="/img/foundations/typography-textstyles.png"
+  srcDark="/img/foundations/typography-textstyles-dark.png"
+  alt="List of all text styles in Salt and how they compare to what is used in UI toolkit"
```

Comment:
@jake-costa I don't think this one is accurate. The text styles being compared are between the brand and legacy themes